package.json

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "main": "index",
   "module": "index.mjs",
+  "types": "index.d.ts",
```

Comment:
Since we will ship `.d.ts` alongside of `.js` files, why do we need this?

Reply:
@JacksonKearl Thanks for link now it make sense.
Can you please create PR against `master`?